lib/Service/src/SettingsHandler.cpp

```diff
@@ -113,6 +117,67 @@ bool SettingsHandler::loadConfigurationFile(const String& filename)
                 m_mqttPort = jsonMqttPort.as<uint16_t>();
             }
 
+            if (false == jsonApSSID.isNull())
+            {
+                m_apSSID = jsonApSSID.as<const char*>();

```

Comment:
const makes no sense ... same in the other assignements.

Reply:
ArduinoJSON does not support `jsonObject.as<char*>`, that is why all assignements are const

Reply:
Oh, I see. Didn't expect that.